lib/perl/Genome/VariantReporting/Command/Wrappers/GoldEvaluation.pm

```diff
@@ -86,5 +88,89 @@ sub is_single_bam {
     return (!defined($model->normal_sample));
 }
 
+sub get_common_translations {
```

Comment:
This is again an artifact from my refactoring to get the module to simply"work".  I initially copied the execute and some supporting logic from `Genome::VariantReporting::Command::Wrappers::ModelReport`.  `get_common_translations` was one of the support methods.  The test was initially created to test the new `get_common_translations` code changes I placed into this module.  Along the path to getting things working,  the `common_translations` method got refactored into the broken down `get_somatic_translations` and `get_germline_translations`.  

I left the `get_common_translations` method around, to keep the "framework" symmetry with the other variant reporting wrapper modules.